[msx] test Started org.apache.hadoop.hdfs.web.TestWebHDFS#testExceptionPropogationInAbstractRunner
[msx] unitTestCounterInClass = 0
2020-03-14 08:25:53,317 [main] INFO  namenode.NameNode (NameNode.java:<clinit>(931)) - [msx-restart] reconfPoint = 0
2020-03-14 08:25:53,992 [main] INFO  namenode.NameNode (NameNode.java:performReconfig(958)) - [msx-restart] NameNode init, reconfigMode is none, do nothing
2020-03-14 08:25:54,979 [main] INFO  namenode.NameNode (NameNode.java:<init>(1042)) - [msx-restart] NameNode 35408 start
2020-03-14 08:25:54,994 [main] INFO  datanode.DataNode (DataNode.java:<clinit>(433)) - [msx-restart] reconfPoint = 0
2020-03-14 08:25:55,057 [main] INFO  datanode.DataNode (DataNode.java:performReconfig(460)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2020-03-14 08:25:55,347 [main] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2750)) - [msx-restart] DataNode 45964 start
2020-03-14 08:25:58,112 [main] INFO  datanode.DataNode (DataNode.java:shutdown(2075)) - [msx-restart] DataNode 45964 stop, double check before stop, dfs.data.transfer.client.tcpnodelay = true
2020-03-14 08:25:58,277 [main] INFO  namenode.NameNode (NameNode.java:stop(1087)) - [msx-restart] NameNode 35408 stop, double check before stop, dfs.data.transfer.client.tcpnodelay = true
[msx] test Finished org.apache.hadoop.hdfs.web.TestWebHDFS#testExceptionPropogationInAbstractRunner
[msx] writeFile testName = org.apache.hadoop.hdfs.web.TestWebHDFS#testExceptionPropogationInAbstractRunner
[msx] succeed
2020-03-14 08:25:58,340 [main] INFO  server.JournalNode (JournalNode.java:<clinit>(168)) - [msx-restart] reconfPoint = 0
[msx] reset performed
[msx] all testRunFinished
